File: backend/internal/domain/report.go
package domain

import "time"

type Report struct {
	ID          string    `json:"id"`
	WorkshopID  string    `json:"workshop_id"`
	EntryID     string    `json:"entry_id"`
	VehicleID   string    `json:"vehicle_id"`
	Type        string    `json:"type"` // inspection, repair, full
	PDFURL      string    `json:"pdf_url"`
	GeneratedAt time.Time `json:"generated_at"`
	CreatedAt   time.Time `json:"created_at"`
}

File: packages/types/src/report.ts
export interface Report {
  id: string;
  workshop_id: string;
  entry_id: string;
  vehicle_id: string;
  type: 'inspection' | 'repair' | 'full';
  pdf_url: string;
  generated_at: string;
  created_at: string;
}
